packages/react-table/src/components/Table/ActionsColumn.tsx

```diff
@@ -38,7 +38,7 @@ const ActionsColumnBase: React.FunctionComponent<ActionsColumnProps> = ({
   popperProps = {
     position: 'right',
     direction: 'down',
-    popperMatchesTriggerWidth: false
+    minWidth: 'fit-content'
```

Comment:
is this needed? shouldn't the min-width of the trigger allow this to grow as bit as its needed?